src/audio/module_adapter/Kconfig

```diff
@@ -184,6 +184,42 @@ config COMP_PEAK_VOL
          This option enables reporting to host peak vol regs.
          See: struct ipc4_peak_volume_regs
 
+choice "PEAK_METER_UPDATE_PERIOD_CHOICE"
+	prompt "The periods(ms) of updating peak peter value"
```

Comment:
Peak Peter is nice and alliterative but we should fix this typo I think , both here and the commit message

Reply:
thanks, done!